_data/conferences.yml

```
@@ -256,7 +256,7 @@
   date: October 7-9, 2024
   start: 2024-10-07
   end: 2024-10-09
-  sub: NLP, ML
+  sub: ML
```

In case it's useful, note that another solution (which I tried locally and worked) is to convert the field from a string into a list (it's currently a YAML string):

```suggestion
  sub: ['NLP', 'ML']
```